Package not found error - arm64

[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

<!--- If you are new to Docker or this application our issue tracker is **ONLY** used for reporting bugs or requesting features. Please use [our discord server](https://discord.gg/YWrKVTn) for general support. --->

<!--- If this acts as a feature request please ask yourself if this modification is something the whole userbase will benefit from --->
<!--- If this is a specific change for corner case functionality or plugins please look at making a Docker Mod or local script  https://blog.linuxserver.io/2019/09/14/customizing-our-containers/ -->

<!--- Provide a general summary of the issue in the Title above -->

------------------------------

## Expected Behavior
Finds package, works

## Current Behavior
Shows error page.

Logs:

```
Error: Cannot find module 'yauzl',
Require stack:,
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:831:15),
  ],,
    at /usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:14:10,
    at Function.Module._load (internal/modules/cjs/loader.js:687:27),
    at Module.require (internal/modules/cjs/loader.js:903:19),
    at require (internal/modules/cjs/helpers.js:74:18),
    at t.load (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:17:960),
    at e.load (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:13:941),
    at o (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:31:189),
    at Object.errorback (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:31:313),
    at e.triggerErrorback (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:14:306),
    at ReadFileContext.callback (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:18:342),
    at FSReqCallback.readFileAfterOpen [as oncomplete] (fs.js:261:13) {,
  requireStack: [,
    '/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/bootstrap-amd.js',,
    '/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/server/fork.js',
  phase: 'loading',,
  moduleId: 'yauzl',,
  neededBy: [ 'vs/base/node/zip' ],
}
```

## Steps to Reproduce
Install docker image (rpi4 ubuntu arm64)
Connect to web page.


## Environment
**OS: Ubuntu Server**
**CPU architecture:** arm64
**How docker service was installed:**
Installed, docker repo

## Command used to create docker container (run/create/compose/screenshot)

Compose snippet
```
  code-server:
    image: linuxserver/code-server
    container_name: code-server
    networks:
      - backend
    env_file: 
      - "./conf/code-server.env"
    volumes:
      - ./data/vscode-server/config:/config
      - .:/config/workspace
      - ../.ssh:/config/.ssh
    restart: always
    healthcheck:
      test: pidof node || exit 1
      interval: 120s
      timeout: 10s
      retries: 3
```

## Docker logs
```
code-server           | [s6-init] making user provided files available at /var/run/s6/etc...exited 0.
code-server           | [s6-init] ensuring user provided files have correct perms...exited 0.
code-server           | [fix-attrs.d] applying ownership & permissions fixes...
code-server           | [fix-attrs.d] done.
code-server           | [cont-init.d] executing container initialization scripts...
code-server           | [cont-init.d] 01-envfile: executing... 
code-server           | [cont-init.d] 01-envfile: exited 0.
code-server           | [cont-init.d] 10-adduser: executing... 
code-server           | 
code-server           | -------------------------------------
code-server           |           _         ()
code-server           |          | |  ___   _    __
code-server           |          | | / __| | |  /  \ 
code-server           |          | | \__ \ | | | () |
code-server           |          |_| |___/ |_|  \__/
code-server           | 
code-server           | 
code-server           | Brought to you by linuxserver.io
code-server           | -------------------------------------
code-server           | 
code-server           | To support LSIO projects visit:
code-server           | https://www.linuxserver.io/donate/
code-server           | -------------------------------------
code-server           | GID/UID
code-server           | -------------------------------------
code-server           | 
code-server           | User uid:    1001
code-server           | User gid:    1001
code-server           | -------------------------------------
code-server           | 
code-server           | [cont-init.d] 10-adduser: exited 0.
code-server           | [cont-init.d] 30-config: executing... 
code-server           | setting up sudo access
code-server           | adding abc to sudoers
code-server           | setting sudo password
code-server           | Enter new UNIX password: Retype new UNIX password: passwd: password updated successfully
code-server           | [cont-init.d] 30-config: exited 0.
code-server           | [cont-init.d] 99-custom-scripts: executing... 
code-server           | [custom-init] no custom files found exiting...
code-server           | [cont-init.d] 99-custom-scripts: exited 0.
code-server           | [cont-init.d] done.
code-server           | [services.d] starting services
code-server           | [services.d] done.
code-server           | [2020-11-17T04:49:46.390Z] info  Using user-data-dir ~/data
code-server           | [2020-11-17T04:49:46.422Z] info  code-server 3.6.2 9bde62fbd611a7a91c5f327fa43e0d06f1379169
code-server           | [2020-11-17T04:49:46.422Z] info  Using config file ~/.config/code-server/config.yaml
code-server           | [2020-11-17T04:49:46.438Z] info  HTTP server listening on http://0.0.0.0:8443
code-server           | [2020-11-17T04:49:46.438Z] info      - Using password from $PASSWORD
code-server           | [2020-11-17T04:49:46.438Z] info      - To disable use `--auth none`
code-server           | [2020-11-17T04:49:46.439Z] info    - Not serving HTTPS
code-server           | [2020-11-17T04:49:46.439Z] info    - Proxying the following domain:
code-server           | [2020-11-17T04:49:46.440Z] info      - *.www.winters.nz
code-server           | Error: Cannot find module 'semver-umd'
code-server           | Require stack:
code-server           | - /usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/bootstrap-amd.js
code-server           | - /usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/server/fork.js
code-server           |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:831:15)
code-server           |     at Function.Module._load (internal/modules/cjs/loader.js:687:27)
code-server           |     at Module.require (internal/modules/cjs/loader.js:903:19)
code-server           |     at require (internal/modules/cjs/helpers.js:74:18)
code-server           |     at t.load (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:17:960)
code-server           |     at e.load (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:13:941)
code-server           |     at o (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:31:189)
code-server           |     at Object.errorback (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:31:313)
code-server           |     at e.triggerErrorback (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:14:306)
code-server           |     at /usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:14:10
code-server           |     at ReadFileContext.callback (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:18:342)
code-server           |     at FSReqCallback.readFileAfterOpen [as oncomplete] (fs.js:261:13) {
code-server           |   code: 'MODULE_NOT_FOUND',
code-server           |   requireStack: [
code-server           |     '/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/bootstrap-amd.js',
code-server           |     '/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/server/fork.js'
code-server           |   ],
code-server           |   phase: 'loading',
code-server           |   moduleId: 'semver-umd',
code-server           |   neededBy: [
code-server           |     'vs/platform/extensionManagement/node/extensionDownloader',
code-server           |     'vs/workbench/services/extensions/node/extensionPoints',
code-server           |     'vs/platform/extensionManagement/node/extensionsScanner',
code-server           |     'vs/platform/extensionManagement/node/extensionManagementService'
code-server           |   ]
code-server           | }
code-server           | Error: Cannot find module 'applicationinsights'
code-server           | Require stack:
code-server           | - /usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/bootstrap-amd.js
code-server           | - /usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/server/fork.js
code-server           |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:831:15)
code-server           |     at Function.Module._load (internal/modules/cjs/loader.js:687:27)
code-server           |     at Module.require (internal/modules/cjs/loader.js:903:19)
code-server           |     at require (internal/modules/cjs/helpers.js:74:18)
code-server           |     at t.load (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:17:960)
code-server           |     at e.load (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:13:941)
code-server           |     at o (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:31:189)
code-server           |     at Object.errorback (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:31:313)
code-server           |     at e.triggerErrorback (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:14:306)
code-server           |     at /usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:14:10
code-server           |     at ReadFileContext.callback (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:18:342)
code-server           |     at FSReqCallback.readFileAfterOpen [as oncomplete] (fs.js:261:13) {
code-server           |   code: 'MODULE_NOT_FOUND',
code-server           |   requireStack: [
code-server           |     '/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/bootstrap-amd.js',
code-server           |     '/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/server/fork.js'
code-server           |   ],
code-server           |   phase: 'loading',
code-server           |   moduleId: 'applicationinsights',
code-server           |   neededBy: [ 'vs/platform/telemetry/node/appInsightsAppender' ]
code-server           | }
code-server           | Error: Cannot find module '@coder/node-browser'
code-server           | Require stack:
code-server           | - /usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/bootstrap-amd.js
code-server           | - /usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/server/fork.js
code-server           |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:831:15)
code-server           |     at Function.Module._load (internal/modules/cjs/loader.js:687:27)
code-server           |     at Module.require (internal/modules/cjs/loader.js:903:19)
code-server           |     at require (internal/modules/cjs/helpers.js:74:18)
code-server           |     at t.load (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:17:960)
code-server           |     at e.load (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:13:941)
code-server           |     at o (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:31:189)
code-server           |     at Object.errorback (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:31:313)
code-server           |     at e.triggerErrorback (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:14:306)
code-server           |     at /usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:14:10
code-server           |     at ReadFileContext.callback (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:18:342)
code-server           |     at FSReqCallback.readFileAfterOpen [as oncomplete] (fs.js:261:13) {
code-server           |   code: 'MODULE_NOT_FOUND',
code-server           |   requireStack: [
code-server           |     '/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/bootstrap-amd.js',
code-server           |     '/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/server/fork.js'
code-server           |   ],
code-server           |   phase: 'loading',
code-server           |   moduleId: '@coder/node-browser',
code-server           |   neededBy: [ 'vs/server/node/channel' ]
code-server           | }
code-server           | Error: Cannot find module 'yazl'
code-server           | Require stack:
code-server           | - /usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/bootstrap-amd.js
code-server           | - /usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/server/fork.js
code-server           |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:831:15)
code-server           |     at Function.Module._load (internal/modules/cjs/loader.js:687:27)
code-server           |     at Module.require (internal/modules/cjs/loader.js:903:19)
code-server           |     at require (internal/modules/cjs/helpers.js:74:18)
code-server           |     at t.load (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:17:960)
code-server           |     at e.load (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:13:941)
code-server           |     at o (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:31:189)
code-server           |     at Object.errorback (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:31:313)
code-server           |     at e.triggerErrorback (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:14:306)
code-server           |     at /usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:14:10
code-server           |     at ReadFileContext.callback (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:18:342)
code-server           |     at FSReqCallback.readFileAfterOpen [as oncomplete] (fs.js:261:13) {
code-server           |   code: 'MODULE_NOT_FOUND',
code-server           |   requireStack: [
code-server           |     '/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/bootstrap-amd.js',
code-server           |     '/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/server/fork.js'
code-server           |   ],
code-server           |   phase: 'loading',
code-server           |   moduleId: 'yazl',
code-server           |   neededBy: [ 'vs/base/node/zip' ]
code-server           | }
code-server           | Error: Cannot find module 'yauzl'
code-server           | Require stack:
code-server           | - /usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/bootstrap-amd.js
code-server           | - /usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/server/fork.js
code-server           |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:831:15)
code-server           |     at Function.Module._load (internal/modules/cjs/loader.js:687:27)
code-server           |     at Module.require (internal/modules/cjs/loader.js:903:19)
code-server           |     at require (internal/modules/cjs/helpers.js:74:18)
code-server           |     at t.load (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:17:960)
code-server           |     at e.load (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:13:941)
code-server           |     at o (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:31:189)
code-server           |     at Object.errorback (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:31:313)
code-server           |     at e.triggerErrorback (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:14:306)
code-server           |     at /usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:14:10
code-server           |     at ReadFileContext.callback (/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/loader.js:18:342)
code-server           |     at FSReqCallback.readFileAfterOpen [as oncomplete] (fs.js:261:13) {
code-server           |   code: 'MODULE_NOT_FOUND',
code-server           |   requireStack: [
code-server           |     '/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/bootstrap-amd.js',
code-server           |     '/usr/local/share/.config/yarn/global/node_modules/code-server/lib/vscode/out/vs/server/fork.js'
code-server           |   ],
code-server           |   phase: 'loading',
code-server           |   moduleId: 'yauzl',
code-server           |   neededBy: [ 'vs/base/node/zip' ]
code-server           | }
code-server           | [2020-11-17T04:52:07.850Z] error VS Code exited unexpectedly with code 0
```

--- a/.github/workflows/external_trigger.yml
+++ b/.github/workflows/external_trigger.yml
@@ -19,7 +19,7 @@ jobs:
           echo "**** External trigger running off of master branch. To disable this trigger, set a Github secret named \"PAUSE_EXTERNAL_TRIGGER_CODE_SERVER_MASTER\". ****"
           echo "**** Retrieving external version ****"
           EXT_RELEASE=$(curl -u "${{ secrets.CR_USER }}:${{ secrets.CR_PAT }}" -sX GET "https://api.github.com/repos/cdr/code-server/releases/latest" | jq -r '. | .tag_name')
-          if [ -z "${EXT_RELEASE}" ]; then
+          if [ -z "${EXT_RELEASE}" ] || [ "${EXT_RELEASE}" == "null" ]; then
             echo "**** Can't retrieve external version, exiting ****"
             FAILURE_REASON="Can't retrieve external version for code-server branch master"
             curl -X POST -H "Content-Type: application/json" --data '{"avatar_url": "https://cdn.discordapp.com/avatars/354986384542662657/df91181b3f1cf0ef1592fbe18e0962d7.png","embeds": [{"color": 16711680,
@@ -62,17 +62,21 @@ jobs:
           if [ "${EXT_RELEASE}" == "${IMAGE_VERSION}" ]; then
             echo "**** Version ${EXT_RELEASE} already pushed, exiting ****"
             exit 0
+          elif [ $(curl -s https://ci.linuxserver.io/job/Docker-Pipeline-Builders/job/docker-code-server/job/master/lastBuild/api/json | jq -r '.building') == "true" ]; then
+            echo "**** New version ${EXT_RELEASE} found; but there already seems to be an active build on Jenkins; exiting ****"
+            exit 0
           else
             echo "**** New version ${EXT_RELEASE} found; old version was ${IMAGE_VERSION}. Triggering new build ****"
             response=$(curl -iX POST \
               https://ci.linuxserver.io/job/Docker-Pipeline-Builders/job/docker-code-server/job/master/buildWithParameters?PACKAGE_CHECK=false \
               --user ${{ secrets.JENKINS_USER }}:${{ secrets.JENKINS_TOKEN }} | grep -i location | sed "s|^[L|l]ocation: \(.*\)|\1|")
-            echo "**** Jenkins job queue url: ${response} ****"
+            echo "**** Jenkins job queue url: ${response%$'\r'} ****"
             echo "**** Sleeping 10 seconds until job starts ****"
             sleep 10
             buildurl=$(curl -s "${response%$'\r'}api/json" | jq -r '.executable.url')
             buildurl="${buildurl%$'\r'}"
-            echo "**** Jenkins job build url: ${buildurl}; attempting to change the job description ****"
+            echo "**** Jenkins job build url: ${buildurl} ****"
+            echo "**** Attempting to change the Jenkins job description ****"
             curl -iX POST \
               "${buildurl}submitDescription" \
               --user ${{ secrets.JENKINS_USER }}:${{ secrets.JENKINS_TOKEN }} \

--- a/.github/workflows/package_trigger.yml
+++ b/.github/workflows/package_trigger.yml
@@ -16,16 +16,21 @@ jobs:
             echo "**** Github secret PAUSE_PACKAGE_TRIGGER_CODE_SERVER_MASTER is set; skipping trigger. ****"
             exit 0
           fi
+          if [ $(curl -s https://ci.linuxserver.io/job/Docker-Pipeline-Builders/job/docker-code-server/job/master/lastBuild/api/json | jq -r '.building') == "true" ]; then
+            echo "**** There already seems to be an active build on Jenkins; skipping package trigger ****"
+            exit 0
+          fi
           echo "**** Package trigger running off of master branch. To disable, set a Github secret named \"PAUSE_PACKAGE_TRIGGER_CODE_SERVER_MASTER\". ****"
           response=$(curl -iX POST \
             https://ci.linuxserver.io/job/Docker-Pipeline-Builders/job/docker-code-server/job/master/buildWithParameters?PACKAGE_CHECK=true \
             --user ${{ secrets.JENKINS_USER }}:${{ secrets.JENKINS_TOKEN }} | grep -i location | sed "s|^[L|l]ocation: \(.*\)|\1|")
-          echo "**** Jenkins job queue url: ${response} ****"
+          echo "**** Jenkins job queue url: ${response%$'\r'} ****"
           echo "**** Sleeping 10 seconds until job starts ****"
           sleep 10
           buildurl=$(curl -s "${response%$'\r'}api/json" | jq -r '.executable.url')
           buildurl="${buildurl%$'\r'}"
-          echo "**** Jenkins job build url: ${buildurl}; attempting to change the job description ****"
+          echo "**** Jenkins job build url: ${buildurl} ****"
+          echo "**** Attempting to change the Jenkins job description ****"
           curl -iX POST \
             "${buildurl}submitDescription" \
             --user ${{ secrets.JENKINS_USER }}:${{ secrets.JENKINS_TOKEN }} \

--- a/.github/workflows/package_trigger_scheduler.yml
+++ b/.github/workflows/package_trigger_scheduler.yml
@@ -29,6 +29,7 @@ jobs:
               echo "**** Branch ${br} appears to be live; checking workflow. ****"
               if curl -sfX GET https://raw.githubusercontent.com/linuxserver/docker-code-server/${br}/.github/workflows/package_trigger.yml > /dev/null 2>&1; then
                 echo "**** Workflow exists. Triggering package trigger workflow for branch ${br}. ****"
+                triggered_branches="${triggered_branches}${br} "
                 curl -iX POST \
                   -H "Authorization: token ${{ secrets.CR_PAT }}" \
                   -H "Accept: application/vnd.github.v3+json" \
@@ -42,3 +43,8 @@ jobs:
               echo "**** ${br} appears to be a dev branch; skipping trigger. ****"
             fi
           done
+          echo "**** Package check build(s) triggered for branch(es): ${triggered_branches} ****"
+          echo "**** Notifying Discord ****"
+          curl -X POST -H "Content-Type: application/json" --data '{"avatar_url": "https://cdn.discordapp.com/avatars/354986384542662657/df91181b3f1cf0ef1592fbe18e0962d7.png","embeds": [{"color": 9802903,
+            "description": "**Package Check Build(s) Triggered for code-server** \n**Branch(es):** '"${triggered_branches}"' \n**Build URL:** '"https://ci.linuxserver.io/blue/organizations/jenkins/Docker-Pipeline-Builders%2Fdocker-code-server/activity/"' \n"}],
+            "username": "Github Actions"}' ${{ secrets.DISCORD_WEBHOOK }}

--- a/Dockerfile
+++ b/Dockerfile
@@ -45,7 +45,7 @@ RUN \
  fi && \
  CODE_VERSION=$(echo "$CODE_RELEASE" | awk '{print substr($1,2); }') && \
  yarn config set network-timeout 600000 -g && \
- yarn --production --frozen-lockfile global add code-server@"$CODE_VERSION" && \
+ yarn --production --verbose --frozen-lockfile global add code-server@"$CODE_VERSION" && \
  yarn cache clean && \
  echo "**** clean up ****" && \
  apt-get purge --auto-remove -y \

--- a/Dockerfile
+++ b/Dockerfile
@@ -44,6 +44,7 @@ RUN \
 	| awk '/tag_name/{print $4;exit}' FS='[""]'); \
  fi && \
  CODE_VERSION=$(echo "$CODE_RELEASE" | awk '{print substr($1,2); }') && \
+ yarn config set network-timeout 600000 -g && \
  yarn --production --frozen-lockfile global add code-server@"$CODE_VERSION" && \
  yarn cache clean && \
  echo "**** clean up ****" && \

--- a/Dockerfile.aarch64
+++ b/Dockerfile.aarch64
@@ -45,7 +45,7 @@ RUN \
  fi && \
  CODE_VERSION=$(echo "$CODE_RELEASE" | awk '{print substr($1,2); }') && \
  yarn config set network-timeout 600000 -g && \
- yarn --production --frozen-lockfile global add code-server@"$CODE_VERSION" && \
+ yarn --production --verbose --frozen-lockfile global add code-server@"$CODE_VERSION" && \
  yarn cache clean && \
  echo "**** clean up ****" && \
  apt-get purge --auto-remove -y \

--- a/Dockerfile.aarch64
+++ b/Dockerfile.aarch64
@@ -44,6 +44,7 @@ RUN \
 	| awk '/tag_name/{print $4;exit}' FS='[""]'); \
  fi && \
  CODE_VERSION=$(echo "$CODE_RELEASE" | awk '{print substr($1,2); }') && \
+ yarn config set network-timeout 600000 -g && \
  yarn --production --frozen-lockfile global add code-server@"$CODE_VERSION" && \
  yarn cache clean && \
  echo "**** clean up ****" && \

--- a/Dockerfile.armhf
+++ b/Dockerfile.armhf
@@ -45,7 +45,7 @@ RUN \
  fi && \
  CODE_VERSION=$(echo "$CODE_RELEASE" | awk '{print substr($1,2); }') && \
  yarn config set network-timeout 600000 -g && \
- yarn --production --frozen-lockfile global add code-server@"$CODE_VERSION" && \
+ yarn --production --verbose --frozen-lockfile global add code-server@"$CODE_VERSION" && \
  yarn cache clean && \
  echo "**** clean up ****" && \
  apt-get purge --auto-remove -y \

--- a/Dockerfile.armhf
+++ b/Dockerfile.armhf
@@ -44,6 +44,7 @@ RUN \
 	| awk '/tag_name/{print $4;exit}' FS='[""]'); \
  fi && \
  CODE_VERSION=$(echo "$CODE_RELEASE" | awk '{print substr($1,2); }') && \
+ yarn config set network-timeout 600000 -g && \
  yarn --production --frozen-lockfile global add code-server@"$CODE_VERSION" && \
  yarn cache clean && \
  echo "**** clean up ****" && \

